newsplease/crawler/simple_crawler.py

```diff
@@ -1,7 +1,74 @@
+# Based on https://github.com/adbar/trafilatura/blob/master/trafilatura/utils.py
+import logging
+import socket
 import copy
 import threading
 
-from six.moves import urllib
+try:
+    # this module is faster
+    import cchardet
+except ImportError:
+    cchardet = None
+# https://charset-normalizer.readthedocs.io/en/latest/
+# https://ftfy.readthedocs.io/en/latest/
+
+import requests
+import urllib3
+
+MAX_FILE_SIZE = 20000000
+MIN_FILE_SIZE = 10
+
+LOGGER = logging.getLogger(__name__)
+
+# customize headers
+HEADERS = {
+    'Connection': 'close',  # another way to cover tracks
```

Comment:
please use the user-agent information from news-please's config file rather than have a different one hard coded here

Reply:
Simple_crawler originally used a hard-coded header setting as well. How should I set it to news-please config?

Reply:
Oh, I'm sorry. I confused this with another part of news-please. We can leave it as it is.